src/core/backend_context.cc

```diff
@@ -1241,6 +1254,7 @@ BackendInputCollector::FlushPendingCopyKernel(
     const TRITONSERVER_MemoryType tensor_memory_type,
     const int64_t tensor_memory_type_id)
 {
+  LOG_ERROR << "before launching gather kernel";
```

Comment:
Did you mean to add these temporarily for logging. I think they should be removed.